### PR TITLE
Update SkinJSON.php

### PR DESCRIPTION
**Pull request by LorenMaxwell**
1) Added JSON_PRETTY_PRINT option
2) Added SkinJSONDebug variable to allow turning on/off debugging from skin.json file

### CODE REVIEW

File: SkinJSON.php [hunk 1]
```diff
@@ -47,8 +47,12 @@ function getUser() {
 	 * This makes SkinJSON work with the MobileFrontend ContentProvider proxy.
 	 */
 	public static function onOutputPageBeforeHTML( $out, &$html ) {
-		$out->addModules( [ 'skins.skinjson.debug' ] );
-		$out->addModuleStyles( [ 'skins.skinjson.debug.styles' ] );
+	    global $wgSkinJSONDebug;
```
Comment (jdlrobson) [inline SkinJSON.php]: Please use $out->getConfig()->get('SkinJSONDebug') here

**Review by jdlrobson — CHANGES_REQUESTED** [review]

**Review by LorenMaxwell — COMMENTED** [review]
Updated to $out->getConfig()->get('SkinJSONDebug')

**Review by jdlrobson — APPROVED** [review]

**Comment by jdlrobson** [comment]
Will pull this down to test later but I am not anticipating any problems. Thanks for sending these!

File: SkinJSON.php [hunk 2]
```diff
@@ -46,9 +46,12 @@ function getUser() {
 	 * Forwards OutputPageBeforeHTML hook modifications to the template
 	 * This makes SkinJSON work with the MobileFrontend ContentProvider proxy.
 	 */
-	public static function onOutputPageBeforeHTML( $out, &$html ) {
-		$out->addModules( [ 'skins.skinjson.debug' ] );
-		$out->addModuleStyles( [ 'skins.skinjson.debug.styles' ] );
+	public static function onOutputPageBeforeHTML( $out, &$html )
```
Comment (jdlrobson) [inline SkinJSON.php]: Missing a brace here {

**Review by jdlrobson — APPROVED** [review]

**Pull request merged**